### ISSUE TITLE
AttributeError: 'tuple' object has no attribute 'get' in security wrapper error path

### ISSUE THREAD

**Issue by mmattb** [zalando/connexion#1313]
### Description
Connexion doesn't handle error in a security path. Specifically, it expects a dictionary-like object but is getting a tuple with an err msg and response code.

```
Traceback (most recent call last):
  File "/root/.local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/root/.local/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/root/.local/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/root/.local/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/root/.local/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/root/.local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/root/.local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/root/.local/lib/python3.8/site-packages/connexion/decorators/decorator.py", line 48, in wrapper
    response = function(request)
  File "/root/.local/lib/python3.8/site-packages/connexion/decorators/security.py", line 326, in wrapper
    request.context['user'] = token_info.get('sub', token_info.get('uid'))
AttributeError: 'tuple' object has no attribute 'get'
```

### Expected behaviour
The error should be caught and bubbled up in clear fashion by an exception, rather than exploding on this AttributeError. It would be much better to bubble up the error msg in an exception, or return the error msg.

### Actual behaviour
A difficult-to-understand exception.  The User doesn't know what to fix here.

### Steps to reproduce
I'm not sure why this is happening yet, so I can't say precisely what is causing the error to bubble up.

### Additional info:
Flask version: 1.1.2

Output of the commands:

- `python --version`
Python 3.8.2

- `pip show connexion | grep "^Version\:"`
Version: 2.7.0

**Comment by instantlinux** [comment]
This is probably a dup of #1268. I encountered the same symptom when I added a second security scheme thus:
```
  securitySchemes:
    basic:
      type: http
      scheme: basic
      x-basicInfoFunc: apicrud.session_auth.basic
    apikey:
      in: header
      name: X-API-KEY
      type: apiKey
      x-apikeyInfoFunc: apicrud.session_auth.api_key
```
This exception gets hit on the path through `apiKey` logic if your session_auth function returns what's typically returned for flask (a tuple containing two values, a dict and an http status code). As a workaround, I'm eliminating the status_code response for my api_key() function and instead of returning (say) a tuple with `(dict(message='Access denied'), 403)`, for now it will just return `None` without any message.
